torch/_dynamo/callback.py

```diff
@@ -1,62 +1,66 @@
-# mypy: allow-untyped-defs
+from typing import Callable, List
+
+
 class CompilationCallbackHandler:
```

Comment:
This would definitely be a good candidate to convert to a Dataclass, it just holds two typed lists.

Reply:
I put up a draft diff here: https://github.com/pytorch/pytorch/pull/137312

Will publish when CI turns green.